### PR TITLE
Add logging flags for remotefs

### PR DESCRIPTION
**Pull request by jhowardmsft**
Signed-off-by: John Howard <jhoward@microsoft.com>

@jterry75 Don't ask. I really don't like it, but it works.

### CODE REVIEW

File: service/gcsutils/gcstools/commoncli/common.go [hunk 1]
```diff
@@ -70,19 +71,36 @@ func SetupTar2VHDLibOptions(args ...*string) (*libtar2vhd.Options, error) {
...
+	logrus.SetLevel(level)
+
+	// Add the sub-command to the filename for remotefs. This is really ugly :(
+	// Done to allow mixing both fixed args that remotefs expects for back-compat,
+	// along with flags. Prior to this change, remotefs didn't accept flags at all,
+	// and did no logging. Unfortunately, there are circumstances where two distinct
+	// remotefs commands can execute simultaneously, so we need separate logfiles.
+	filename := *args[0]
+	if os.Args[0] == "remotefs" {
+		if len(flag.Args()[0]) > 1 {
```
Comment (jterry75) [inline service/gcsutils/gcstools/commoncli/common.go]: Oh wow.

**Comment by jterry75** [comment]
I hesitate to say LGTM but LGTM.

**Comment by gupta-ak** [comment]
Hm, couldn't the flags be added to the end of the remotefs cmdline? Like the format would look: `remotefs <CMD> [CMD ARGS] [flags]`. For example, the code sequence for `remotefs stat <path>` just checks if the len(argv) <= 3, so I think it won't reject something like `remotefs stat <path> --logging=debug`.

**Comment by jhowardmsft** [comment]
@gupta-ak No. flags.parse expects its flags to be first unfortunately. When it gets down to the functions, the arguments passed are just those which aren't flags, so the current logic still works.

**Comment by gupta-ak** [comment]
Oh I see. It works differently than `getopt` which lets you put the flags at the end. That is unfortunate :(

**Comment by jhowardmsft** [comment]
So OK to merge?

**Pull request merged**